src/main/java/com/dawsonsystems/session/MongoManager.java

```diff
@@ -269,7 +271,7 @@ public org.apache.catalina.Session createSession() {
   public org.apache.catalina.Session createSession(java.lang.String sessionId) {
     StandardSession session = (MongoSession) createEmptySession();
 
-    log.fine("Created session with id " + session.getIdInternal() + " ( " + sessionId + ")");
+    log.fine("Created session with id " + session.getIdInternal() + " ( " + clipSessionId(sessionId) + ")");
```

Comment:
Use the `Supplier` overload to avoid clipping and concatenation when level is not set to FINE